.pylintrc

```diff
@@ -99,7 +99,7 @@ evaluation=10.0 - ((float(5 * error + warning + refactor + convention) / stateme
 [FORMAT]
 
 # Maximum number of characters on a single line.
-max-line-length=79
+max-line-length=120
```

Comment:
I would prefer sticking with the default line length, because that works nicely with the side by side view in github pull requests. However, if you have a strong opinion on this, I don't mind increasing it..